actionpack/test/dispatch/request/json_params_parsing_test.rb

```diff
@@ -37,9 +37,9 @@ def teardown
     )
   end
 
-  test "parses json params for application/vnd.api+json" do
+  test "does not parses unregistered media types such as application/vnd.api+json" do
```

Comment:
@rafaelfranca typo `does not parses` should be `does not parse`.  Just leave it for someone else?